.github/workflows/test.yml

```
@@ -0,0 +1,58 @@
+name: Grunt tests
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
```

Should we try to rely more on defaults here, like in https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-a-list-of-events:
```suggestion
on: [push, pull_request]
```
In particular, the `edited` event seems to run when a review is edited if I understand it correctly. Docs are pretty scarce on what each event means, unfortunately...

---

I copied this from one of my projects and afair I did this so tags don't trigger a run as my project had much more jobs and was "expensive". Let's go with the default and change it if needed. I've changed this to your suggestion.